web/docs/advanced/jobs.md

```diff
@@ -416,11 +416,10 @@ The return value of `submit()` is an instance of `SubmittedJob`, which has the f
 - `jobId`: The ID for the job in that executor.
 - `jobName`: The name of the job you used in your `.wasp` file.
 - `executorName`: The Symbol of the name of the job executor.
-  - For pg-boss, you can import a Symbol from: `import { PG_BOSS_EXECUTOR_NAME } from '@wasp/jobs/core/pgBoss/pgBossJob.js'` if you wish to compare against `executorName`.
```

Comment:
I just removed this for now -> it is not important until we don't have multiple job exeuctors, so no need to export this API at the moment. We can cross this bridge once we get there.